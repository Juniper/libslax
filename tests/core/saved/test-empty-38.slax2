version 1.2;

param $host = "localhost";

main <data> {
    <val> slax:ends-with("testing", "ing");
    <val> slax:ends-with("testing", "ingg");
    <val> slax:ends-with("testing", "g");
    <val> slax:ends-with("testing", "n");
    var $x = slax:get-host($host);
    <localhost> {
        copy-of $x;
    }
    var $y = slax:get-host($x/address[1]);
    <one-twenty-seven> {
        copy-of $y;
    }
}
