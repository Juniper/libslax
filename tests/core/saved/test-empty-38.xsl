<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:slax="http://xml.libslax.org/slax" version="1.0" extension-element-prefixes="slax">
  <xsl:param name="host" select="&quot;localhost&quot;"/>
  <xsl:template match="/">
    <data>
      <val>
        <xsl:value-of xmlns:slax="http://xml.libslax.org/slax" select="slax:ends-with(&quot;testing&quot;, &quot;ing&quot;)"/>
      </val>
      <val>
        <xsl:value-of xmlns:slax="http://xml.libslax.org/slax" select="slax:ends-with(&quot;testing&quot;, &quot;ingg&quot;)"/>
      </val>
      <val>
        <xsl:value-of xmlns:slax="http://xml.libslax.org/slax" select="slax:ends-with(&quot;testing&quot;, &quot;g&quot;)"/>
      </val>
      <val>
        <xsl:value-of xmlns:slax="http://xml.libslax.org/slax" select="slax:ends-with(&quot;testing&quot;, &quot;n&quot;)"/>
      </val>
      <xsl:variable xmlns:slax="http://xml.libslax.org/slax" name="x" select="slax:get-host($host)"/>
      <localhost>
        <xsl:copy-of select="$x"/>
      </localhost>
      <xsl:variable xmlns:slax="http://xml.libslax.org/slax" name="y" select="slax:get-host($x/address[1])"/>
      <one-twenty-seven>
        <xsl:copy-of select="$y/hostname"/>
        <xsl:copy-of select="$y/address-family"/>
        <xsl:copy-of select="$y/address[1]"/>
      </one-twenty-seven>
    </data>
  </xsl:template>
</xsl:stylesheet>
