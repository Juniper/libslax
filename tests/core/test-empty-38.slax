version 1.2;

param $host = "localhost";

main <data> {
    var $x = slax:get-host($host);
    <localhost> {
        copy-of $x;
    }
    var $y = slax:get-host($x/address[1]);
    <one-twenty-seven> {
        copy-of $y;
    }
}
