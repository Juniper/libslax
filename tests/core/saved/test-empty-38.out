<?xml version="1.0"?>
<data>
  <localhost>
    <host>
      <hostname>localhost</hostname>
      <address-family>inet</address-family>
      <address>127.0.0.1</address>
    </host>
  </localhost>
  <one-twenty-seven>
    <host>
      <hostname>localhost</hostname>
      <alias>1.0.0.127.in-addr.arpa</alias>
      <address-family>inet</address-family>
      <address>127.0.0.1</address>
    </host>
  </one-twenty-seven>
</data>
